perftracker/models/regression.py

```diff
@@ -74,16 +75,18 @@ def ptOnJobSave(job):
 
         if r is None:
             regression_tag = job.regression_tag if job.regression_tag else "regression"
-            r = RegressionModel(title="%s regression" % job.project.name, tag=regression_tag, project=job.project, jobs=0)
+            r = RegressionModel(title="%s regression" % job.project.name, tag=regression_tag, project=job.project, jobs=1)
```

Comment:
why 1?

Reply:
Because this method is called from JobModel.save() when we save job linked to this new regression. So, the number of jobs in this regression from the start is 1.